web-client/src/presenter/actions/setProgressForFileUploadAction.ts

```diff
@@ -0,0 +1,79 @@
+import { FileUploadProgressMapType } from '../../../../shared/src/business/entities/EntityConstants';
+import { state } from '@web-client/presenter/app.cerebral';
+
+export const setProgressForFileUploadAction = ({
+  props,
+  store,
+}: ActionProps<{
+  files: {
+    size: string;
+    name: string;
+  };
+}>): {
+  fileUploadProgressMap: FileUploadProgressMapType;
+} => {
+  const { files } = props;
+  const loadedAmounts: Record<string, number> = {};
+  const startTime = new Date();
```

Comment:
can you just add the linting rule to this line, too.

Reply:
sure! though I was wondering, since this is prohibited, should we use DateHandler instead? I was wondering why it's "O.K. to use Date constructor for calculating time duration."

Reply:
Fair question. While I think this implementation is simple and straightforward enough to use `Date.getTime()`, which is why I suggested putting the linting exception in place here, perhaps it is best to stay consistent with the currently adopted approach and use a combindation of `createISODateString` and `dateStringsCompared`

Reply:
upon further I think it'd be better to add the linting rule to the line instead of making the change. thanks!